### PR TITLE
Pd okta 545966 smart card icon

### PR DESCRIPTION
**Pull request by praveendurai-okta**
## Description:

Incremental Change. 

Added description and icon to smart card IDP Authenticator during enrollment and verification flow.

## PR Checklist

- [X] Have you verified the basic functionality for this change?
- [X] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [x] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-545966](https://oktainc.atlassian.net/browse/OKTA-545966)

### Reviewers:
@mauriciocastillosilva-okta @indirathirumalaimurugan-okta @valberbenetz-okta 

#### Enrollment 
Screen Shot 2022-11-02 at 10 45 43 AM

#### Verification:
Screen Shot 2022-11-02 at 10 45 57 AM

### Downstream Monolith Build:

https://bacon-go.aue1e.saasure.net/commits?artifact=monolith&branch=d16t-okta-signin-widget-89874bf-63628a8a&page=1&pageSize=6&sha=57082d77a3b634d3a05e8d2ede0706db1f6f3c6a&tab=main

### CODE REVIEW

File: packages/@okta/i18n/src/properties/login_ok_PL.properties [hunk 1]
```diff
@@ -850,6 +850,7 @@ oie.yubikey.description = 》Îñšéŕţ ţĥé ÝûƀîĶéý îñţö å ÛŠ
 oie.yubikey.passcode.label = 》Îñšéŕţ ţĥéñ ţåþ ýöûŕ ÝûƀîĶéý ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.custom.app.authenticator.title = 》Ĝéţ å þûšĥ ñöţîƒîçåţîöñ 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.custom.app.authenticator.description = 》{0} îš åñ åûţĥéñţîçåţöŕ åþþ، îñšţåļļéð öñ ýöûŕ þĥöñé، ûšéð ţö þŕöṽé ýöûŕ îðéñţîţý 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.smartcard.authenticator.description = 》Ûšé å þĥýšîçåļ šɱåŕţ çåŕð، šûçĥ åš ÞÎṼ öŕ ÇÅÇ، ţö šîĝñ îñ ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```
Comment (valberbenetz-okta) [inline packages/@okta/i18n/src/properties/login_ok_PL.properties:853]: Not sure if this was intentional, but don't translations get generated after the PR is merged?

File: playground/mocks/config/responseConfig.js [hunk 2]
```diff
@@ -1069,6 +1069,13 @@ const selectOktaVerifyMethod = {
   ],
 };
 
+const smartCardEnrollOrVerify = {
+  '/idp/idx/introspect': [
+    'authenticator-enroll-smartcard',
+    // 'authenticator-verification-smartcard',
```
Comment (valberbenetz-okta) [inline playground/mocks/config/responseConfig.js:1075]: Did you want to uncomment this for verification?

**Review by valberbenetz-okta — APPROVED** [review]
LGTM. Left a couple of small comments.

File: playground/mocks/config/responseConfig.js (hunk 2 above)
Comment (praveendurai-okta) [reply playground/mocks/config/responseConfig.js:1075]: When we want to test Verify in playground, we can uncomment `authenticator-verification-smartcard` and comment `authenticator-enroll-smartcard`

File: packages/@okta/i18n/src/properties/login_ok_PL.properties (hunk 1 above)
Comment (praveendurai-okta) [reply packages/@okta/i18n/src/properties/login_ok_PL.properties:853]: Yes it is intentional. We need to run [pseudo-loc](https://okta.slack.com/archives/CAZH2MWEL/p1652720088043439?thread_ts=1652719963.457889&cid=CAZH2MWEL) to generate this file else [linting](https://okta.slack.com/archives/CAZH2MWEL/p1652720487091099?thread_ts=1652719963.457889&cid=CAZH2MWEL) will fail.

**Review by mauriciocastillosilva-okta — APPROVED** [review]

**Pull request merged**